internal/blockchain/ethereum/address_resolver.go

```diff
@@ -55,11 +55,18 @@ type addressResolverInserts struct {
 }
 
 func newAddressResolver(ctx context.Context, localConfig config.Section, cacheManager cache.Manager, enableCache bool) (ar *addressResolver, err error) {
+
+	client, err := ffresty.New(ctx, localConfig)
+
+	if err != nil {
+		return nil, err
```

Comment:
Looks like an extra test is needed to retain coverage and cover this branch.

Something like: https://github.com/hyperledger/firefly-signer/blob/bece7de7dc20d55bc99517a8255c2ec93f596581/internal/rpcserver/server_test.go#L63-L75